### ISSUE TITLE
Tidy up tests and utility helpers

### ISSUE THREAD

**Issue by wlthomson** [openmsupply/msupply-rnr#25]
Utility helpers and tests should be better structured for accessibility and modularity.

**Comment by wlthomson** [comment]
Merged into development. Closing.
